packages/main/src/ListItem.js

```diff
@@ -74,15 +74,15 @@ const metadata = {
 		},
 
 		/**
-		 * Used to define the role of the list item.
+		 * Used to define the role of the internal list item.
 		 *
 		 * @private
 		 * @type {string}
 		 * @defaultvalue "listitem"
-		 * @since 1.0.0-rc.9
+		 * @since 1.2.0
 		 *
 		 */
-		role: {
+		accessibleRole: {
```

Comment:
Despite being private, this property was requested by users to customize the role of the list items, so my guess is that it is used outside the framework on application level #2231. 

If we still decide to rename, it is also used as `role` in the `TreeListItem.js` file https://github.com/SAP/ui5-webcomponents/blob/117d86410cd0ba1b43495d0d32bd00db7d3b3fd2/packages/main/src/TreeListItem.js#L294

Reply:
We discussed this and decided for now to leave both role and accessibleRole properties.

Reply:
Great, I'm OK with that!